Adaptor: Static site (.html files) doesn't work

### Qwik Version

0.12.1

### Operating System (or Browser)

MacOS

### Node Version (if applicable)

16.13.1

### Which component is affected?

Starters / CLI

### Expected Behaviour

`npm run ssg` works like documented in README.md

### Actual Behaviour

```
 ~/E/qwik-app  main  npm run ssg
npm ERR! Missing script: "ssg"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/solidevolution/.npm/_logs/2022-11-01T15_02_04_124Z-debug-0.log
```

### Additional Information

`npm run build.server` doesn't work, too

```
 ~/E/qwik-app  main  npm run build.server

> build.server
> vite build -c adaptors/static/vite.config.ts

vite v3.2.1 building SSR bundle for production...
✓ 15 modules transformed.
server/@qwik-city-plan.mjs              9.58 KiB
server/entry.ssr.mjs                    14.68 KiB
server/assets/index.qwik.34c73f2f.mjs   5.36 KiB
[Error: ENOENT: no such file or directory, open '/Users/solidevolution/Entwicklung/qwik-app/dist/service-worker.js'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/solidevolution/Entwicklung/qwik-app/dist/service-worker.js'
}

SSG results
Generated: 2 pages
Duration: 92.0 ms
Average: 46.0 ms per page
```

--- a/packages/qwik-city/buildtime/vite/plugin.ts
+++ b/packages/qwik-city/buildtime/vite/plugin.ts
@@ -235,14 +235,18 @@ export function qwikCity(userOpts?: QwikCityVitePluginOptions): any {
           for (const swEntry of ctx.serviceWorkers) {
             try {
               const swClientDistPath = join(clientOutDir, swEntry.chunkFileName);
-
               const swCode = await fs.promises.readFile(swClientDistPath, 'utf-8');
-              const swCodeUpdate = prependManifestToServiceWorker(ctx, manifest, swCode);
-              if (swCodeUpdate) {
-                await fs.promises.writeFile(swClientDistPath, swCodeUpdate);
+              try {
+                const swCodeUpdate = prependManifestToServiceWorker(ctx, manifest, swCode);
+                if (swCodeUpdate) {
+                  await fs.promises.mkdir(clientOutDir, { recursive: true });
+                  await fs.promises.writeFile(swClientDistPath, swCodeUpdate);
+                }
+              } catch (e2) {
+                console.error(e2);
               }
             } catch (e) {
-              console.error(e);
+              // safe to ignore if a service-worker.js not found
             }
           }
         }

